src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          class="card__img"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        >
+
+        <div class="card__info">
+          <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+        </div>
+
+
+        <div class="card__review">
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review-rating">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-text">
+            Price:
+          </div>
+          <div class="card__price-price">
+            $2,199
+          </div>
+        </div>
+
+        <a
+          href="#"
+          data-qa="hover"
+          class="card__button"
+        >
+          BUY
+        </a>
+
+      </div>
```

Comment:
```suggestion
        </a>
      </div>
```
Don't leave empty lines between parent and child elements